contact-page.css

```diff
@@ -88,31 +78,37 @@ h2 {
     text-align: left;
 }
 
-.contact-info input {
-    top: 0;
-    margin: 0;
-}
-
 .contact-info textarea {
-    width: 790px;
+    width: 100%;
     padding: 30px;
-    resize: none;
 }
 
-#submit-btn {
-    background-color: rgb(130, 140, 214);
+.button {
+    background-color: #616ecc;
     color: white;
+    padding: 1rem 1rem;
     border-radius: 5px;
     font-size: 1.2rem;
-    height: 40px;
-    width: 150px;
-    margin: 10px;
-    position: relative;
-    border: 0px;
+    text-transform: uppercase;
+    cursor: pointer;
+    letter-spacing: 1px;
     animation-name: btn-animation;
     animation-duration: 5s;
     animation-iteration-count: infinite;
-    text-transform: uppercase;
-    cursor: pointer;
-    letter-spacing: 3px;
+}
+
+button :hover {
+    animation-name: btn-animation;
+    animation-duration: 500ms;
+    @keyframes background-color {
+        0% {
+            background-color: #616ecc;
+        }
+        50% {
+            background-color: #661166;
+        }
+        100% {
+            background-color: #fe8bef;
+        }
+    }

```

Comment:
I think you want to use a hover transition. Just add `transition: background 0.2s ease-in-out` to your `button` and change the background colour in the hover state like this: `.button:hover { background-color: #661166; }`